src/Template/.github/workflows/psalm.yml

```diff
@@ -0,0 +1,75 @@
+name: Psalm
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - '**.php'
+      - 'composer.*'
+      - 'psalm*'
+      - '.github/workflows/psalm.yml'
+  push:
+    branches:
+      - develop
+    paths:
+      - '**.php'
+      - 'composer.*'
+      - 'psalm*'
+      - '.github/workflows/psalm.yml'
+
+jobs:
+  build:
+    name: PHP ${{ matrix.php-versions }} Psalm Analysis
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.4', '8.0', '8.1']
```

Comment:
I think only `7.4` is enough.

> Psalm uses the declaration in composer.json if one is present. It will check against the earliest version of PHP that satisfies the declared php dependency
https://psalm.dev/docs/running_psalm/configuration/#phpversion

Reply:
Updated!